keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1153,6 +1155,29 @@ If resources are unprepared when `Deallocate` is called, `Deallocate`
 might need to perform additional actions to correctly deallocate
 resources.
 
+### Management access
+
+A cluster admin or some other more privileged user may want to access a certain
+resource through a new claim and pod while the resource is in use, for example,
+to monitor health or gather statistics.
+
+Normally, resources are allocated exclusively for a claim, so accessing it
+again would be denied. Declaring a claim as "for management access" by setting
+a boolean in the ResourceClaim changes that. Such a claim gets allocated
+without marking the underlying resource as in-use and thus can co-exist with
+other claims that also access the same resource.
+
+It is the responsibility of the user who asks for management access to ensure
+that it's access to that resource does not interfere with real usage. To
+prevent abuse, the default is to deny allocation of management claims. To allow
+that, the namespace must have a [Quota](#quota) object with the
+`AllowManagementAccess` field set to true.
+
+This new boolean alone is not sufficient to deploy a daemon set which requests
+and gets access to all resource instances of a certain type on a node. Some way
+to ask for ">= 1 instance" with no upper bound will be needed. This can be
+added together with support for optional allocation.
```

Comment:
What I have in mind is adding an optional "range" to a request. No range means "exactly one" (current KEP). Adding a range with "min: 1" could be used to ask for all GPUs. A range with "min: 0, max: 1" would be for a resource that is not absolutely required.

I'll do a separate PR with this.